_posts/release/2018-07-18-accumulo-1.9.3.md

```diff
@@ -36,15 +98,152 @@ I0123abc.rf, you are probably at low risk from this bug.
 
 This issue was fixed in pull request [#562]
 
-### ~Another notable item here~ (affects ~versions~)
+### Updated Master Metrics to include FATE metrics.
+
+Added master metrics to provide a snapshot of current FATE operations.  The metrics added:
++ the number of current FATE transactions in progress, 
++ the count of child operations that have occurred on the zookeeper FATE node
++ a count of zookeeper connection errors when the snapshot is taken.  
+
+The number of child operations provides a light-weight surrogate for FATE transaction 
+progression between snapshots. The metrics are controlled with the following properties:
+
+* master.fate.metrics.enabled - default to _false_ preserve current metric reporting
+* master.fate.metrics.min.update.interval - default to _60s_ - there is a hard limit of 10s.
+
+When enabled, the metrics are published to JMX and can optionally be configured using standard 
+hadoop metrics2 configuration files.
+
+### Fixed issues with Native Maps on Fedora
```

Comment:
This isn't quite right. It's not just Fedora. It's newer `libstdc++` implementations. The Fedora versions just happened to have those versions, but this isn't a Fedora issue, and it could mislead people into thinking they are unaffected if it is framed that way.

Rather than describe it as a Fedora issue, it should be described as an issue pertaining to `libstdc++ 8.2 and higher`.